app/models/application_record.rb

```diff
@@ -1,3 +1,22 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def append_url_options(path, options = {})
+    if options[:format] && options[:locale]
+      path = "#{path}.#{options[:locale]}.#{options[:format]}"
+    elsif options[:locale] && options[:locale] != I18n.default_locale
```

Comment:
am I being dim or are `options[:locale]` and `options[:locale]` not the same? what's going on with this line?

Reply:
The first part (before the `&&`) checks `:locale` exists as a key within the `options` hash, whilst the second part compares it's value.

The default value would be `nil`, so I can probably remove the first part.